src/SwaggerProvider.DesignTime/Utils.fs

```diff
@@ -2,12 +2,21 @@ namespace SwaggerProvider.Internal
 
 module SchemaReader =
   open System
+  open System.IO
+  open System.Net
   open System.Net.Http
 
+  let getAbsolutePath (resolutionFolder:string) (schemaPathRaw:string) =
+    let uri = Uri(schemaPathRaw, UriKind.RelativeOrAbsolute)
+    if uri.IsAbsoluteUri then schemaPathRaw
+    elif Path.IsPathRooted schemaPathRaw
+    then Path.Combine(Path.GetPathRoot(resolutionFolder), schemaPathRaw.Substring(1))
+    else Path.Combine(resolutionFolder, schemaPathRaw)
+
   let readSchemaPath (headersStr:string) (schemaPathRaw:string) =
     async {
-        match schemaPathRaw.StartsWith("http", true, null) with
```

Comment:
previoesly this condition worked like if it start with `http` (so this is url, `http` or `https`) then we load using HTTP request otherwise load local file from file system

Reply:
Agree, missed this point. I'll fix

Reply:
Fixed